solver/exporter.go

```diff
@@ -232,7 +237,7 @@ func (e *exporter) ExportTo(ctx context.Context, t CacheExporterTarget, opt Cach
 		for _, dep := range deps {
 			rec, err := dep.CacheKey.Exporter.ExportTo(ctx, t, opt)
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
Is there something to do with this error? (I see it's fully discarded); something to log, or collect in a multi-error (and have a single log or something)?

Reply:
We could log them as warnings but v0.24 didn't log them as well, so as a regression fix I don't think this change is needed.